### DISCUSSION TITLE
I'm getting "Pages on this repository are disabled." in Settings/Pages

### DISCUSSION THREAD

**Discussion by twoi** [community/community#23927] [Other Feature Feedback, Questions, & Ideas]
<p>I created a public repo “<a href="http://username.github.io" rel="noopener nofollow ugc">username.github.io</a>” (with our github username of course) and in its Settings / Pages page, I get this error:<br>
“Pages on this repository are disabled. Please contact your organization administrators”</p>
<p>Back Story:</p>
<p>Initially, I created a repo with the exact capitalization as the github username, i.e. something like UserName…, and then I ran into above error.</p>
<p>However, I then read that it needs to be all lowercase. I changed the repo name, but the error was the same. So I changed the name back to UserName…, deleted the repo, and created a new one with the name username… But the problem persists.</p>
<p>How can I enable Pages for this repo?</p>

**Comment by tajmone** [comment]
<p>Is the repository public or private?</p>

**Reply by mdfarhan0203** [reply]
I try both

**Comment by twoi** [comment]
<p>It is public.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/TRAdEWORKS/tradeworks.github.io">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/TRAdEWORKS/tradeworks.github.io" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/1841686/181080062-5a3ff4be-67aa-4e4b-a9c6-50e5d47c3d96.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e46b649ba25d0599947f08ede5051954625a58a6_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e46b649ba25d0599947f08ede5051954625a58a6_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/4/e46b649ba25d0599947f08ede5051954625a58a6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e46b649ba25d0599947f08ede5051954625a58a6_2_10x10.png"></div>

<h3><a href="https://github.com/TRAdEWORKS/tradeworks.github.io" target="_blank" rel="noopener nofollow ugc">GitHub - TRAdEWORKS/tradeworks.github.io</a></h3>

  <p>Contribute to TRAdEWORKS/tradeworks.github.io development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>(I had deleted it after my post, but have since recreated it - the error is still there).</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>The error you’re seeing relates to an organization level setting under “member privileges”. This is there to prevent users from publishing organization content to the public:</p>
<p><img src="https://user-images.githubusercontent.com/13207348/181080063-033a8030-6fa3-41f3-98da-2193e9004795.png" alt="ScreenShot 2021-08-12 at 13.27.30" data-base62-sha1="3SEfmxoEYd1AOEnubri3cG3Z07e" width="600" height="257"></p>
<p>It sounds like your organization has Public option disabled, which would prevent anyone in the org from publishing a pages site on <code>&lt;username&gt;.github.io</code>.</p>

**Reply by ShayNehmad-RecoLabs** [reply]
Thanks a lot! That was exactly that. 

I wish that "contact your admin" messages would come with "And tell them that in order to fix it, they need to do {insert instructions here}". Oh dear product managers of the world, heed my call!

**Reply by 5022161505** [reply]
Your welcome

**Reply by 5022161505** [reply]
> The error you’re seeing relates to an organization level setting under “member privileges”. This is there to prevent users from publishing organization content to the public:
> 
> <img alt="ScreenShot 2021-08-12 at 13.27.30" width="600" height="257" src="https://user-images.githubusercontent.com/13207348/181080063-033a8030-6fa3-41f3-98da-2193e9004795.png">
> 
> It sounds like your organization has Public option disabled, which would prevent anyone in the org from publishing a pages site on `<username>.github.io`.

**Comment by twoi** [comment]
<p>That was it. Thanks!</p>

**Reply by hillarychesang** [reply]
my github pages are currently dissabled, i am new to github and i wanted to nquire how i can proceed from here! thank you
![screenshotnew](https://user-images.githubusercontent.com/118069197/201501476-90e52b75-852d-4b89-bb24-5cf05ec6b0b4.PNG)

**Comment by githubCrabSubtle** [comment]


**Reply by hillarychesang** [reply]
how do i go about the problem? thanks

**Reply by hillarychesang** [reply]
Can someone please Help me out on this

**Reply by hillarychesang** [reply]
I will be so grateful, and also I would like to know if I can use github on my Android phone

**Reply by asitsonawane** [reply]
> Can someone please Help me out on this

**Access Your Organization's Settings:**

- In the upper-right corner of GitHub, click on your profile photo.
- Select Your organizations.
- Next to your organization, click Settings.

**Update Pages Publication Settings:**

- In the "Access" section of the sidebar, click Member privileges.
- Under "Pages creation," select Public to allow publicly published GitHub Pages sites.

**Comment by fulldecent** [comment]
Here is the full screenshot

<img width="1571" height="1054" alt="Screenshot 2026-07-30 at 14 13 15" src="https://github.com/user-attachments/assets/99a1fdbe-d606-46af-ad01-a487482d3240" />
